paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -453,6 +453,29 @@ class PaparazziPluginTest {
     failureDir.deleteRecursively()
   }
 
+  @Test
+  fun verifyRenderingModesModule() {
+    val fixtureRoot = File("src/test/projects/verify-rendering-modes")
+
+    gradleRunner
+      .withArguments("testDebug", "--stacktrace")
+      .runFixture(fixtureRoot) { build() }
+
+    val snapshotsDir = File(fixtureRoot, "build/reports/paparazzi/images")
+    val snapshots = snapshotsDir.listFiles().apply { sortBy { it.lastModified() }}
+    assertThat(snapshots!!).hasLength(3)
+
+    val goldenImages = listOf(
+      File(fixtureRoot, "src/test/resources/normal.png"),
+      File(fixtureRoot, "src/test/resources/horizontal_scroll.png"),
+      File(fixtureRoot, "src/test/resources/vertical_scroll.png")
+    )
+
+    assertThat(snapshots[0]).isSimilarTo(goldenImages[0]).withDefaultThreshold()
```

Comment:
nit: creating a list only to index it in the assertion makes me think it'd be best to inline?